### DISCUSSION TITLE
New slash commands not listing.

### DISCUSSION THREAD

**Discussion by FlyingTurkman** [discordjs/discord.js#10168] [Q&A]
I am developing a discord bot but when I added a new slash command its not adding to command list. When I kicked bot and added back, its working.

**Comment by imjoaosilva** [comment]
Hello, this could be a cache issue. What you can do is do control + r when you add a new command. This will do a quick refresh which will probably resolve the issue.

**Comment by pxsty0** [comment]
Are you sure that you sent the command to all servers and not to a single server?
